contextualized/dags/torch_notmad/test_data.py

```diff
@@ -0,0 +1,44 @@
+import numpy as np
```

Comment:
TL;DR: This simulation.py and test_data.py won't live in torch_notmad upon release, so it might fit better in test_notmad while we validate these changes.

We need a tests.py for torch NOTMAD to validate our design choices, and eventually we would like a suite of simulation experiments that allow others to easily replicate results. Right now, simulation.py and test_data.py are used to replicate the old NOTMAD results, which achieves a little bit of both. Ideally we would validate design choices for torch-NOTMAD in isolation (e.g. with simple tests like checking for convergence) and do more interesting simulations in a separate experiments directory.

We will also want to remove test_notmad eventually. I'm ok mergeing test_notmad into dev, but we want to remove test_notmad before merging to main and releasing this change since the TF notmad will be depreciated.